hardware_enablement/kmm-kernel-module-management.adoc

```diff
@@ -16,3 +16,6 @@ include::modules/kmm-installation.adoc[leveloffset=+1]
 include::modules/kmm-installing-using-web-console.adoc[leveloffset=+2]
 include::modules/kmm-installing-using-cli.adoc[leveloffset=+2]
 include::modules/kmm-installing-older-versions.adoc[leveloffset=+2]
+include::modules/kmm-deploying-modules.adoc[leveloffset=+1]
+include::modules/kmm-creating-moduleloader-image.adoc[leveloffset=+1]
+include::modules/kmm-using-driver-tool-kit.adoc[leveloffset=+1]
```

Comment:
I am wondering if DTK deserves an entry here? It is a helper to build images, so should it be a section of `kmm-creating-moduleloader-image.adoc`?

Reply:
I've demoted DTK back to a sub-topic within kmm-creating-moduleloader-image.adoc.
We also do not need a separate JIRA ticket (https://issues.redhat.com/browse/TELCODOCS-991) for this DTK info. I've asked for TELCODOCS-991 to be closed